src/Tools/Tools.jsx

```diff
@@ -235,19 +235,20 @@ class Tools extends Component {
                   isExpanded={true}
                 />
               </div>
-              {COMMERCIAL_DATA_ENABLED && userAccountInfo?.payingAccount && (
-                <Tab
-                  id="commercial-tab"
-                  title={t`Commercial`}
-                  enabled={userAccountInfo?.payingAccount}
-                  renderKey={TABS.COMMERCIAL_TAB}
-                >
-                  <div className="commercial-data-wrapper">
-                    <CommercialData userAccountInfo={userAccountInfo || {}} />
-                  </div>
-                </Tab>
-              )}
             </Tab>
+            {/*This check disabled for now. Waiting on role check response from API*/}
+            {/*{COMMERCIAL_DATA_ENABLED && userAccountInfo?.payingAccount && (*/}
+            <Tab
+              id="commercial-data-tab"
+              title={t`Order`}
+              enabled={userAccountInfo?.payingAccount}
```

Comment:
For the development purposes now, I would comment out this `enabled` property, since now only certain logged-in users would have access to this tab - similar to what you wrote in the comment a few lines above